packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/Messages.java

```diff
@@ -2102,6 +2110,124 @@ ArrayList<Object> toList() {
     }
   }
 
+  /**
+   * Pigeon equivalent of the Cap class.
```

Comment:
Instead of saying "the Cap class" consider linking to the source of that class. 

I believe this is the correct class. https://developers.google.com/maps/documentation/android-sdk/reference/com/google/android/libraries/maps/model/Cap

Reply:
Should this link to google map's SDK's Cap? I would have thought a link to the platform interface's Cap would be more appropriate, but do you disagree?

Reply:
I think the fact that you and I were thinking of different classes is a great explanation why "the cap class" is not the right level of documentation.

Reply:
I've linked here to the platform interface Cap, and from a couple other comments in this PR to the Maps SDK's Cap. Let me know if you agree with my choice of links.